setup.py

```diff
@@ -76,6 +76,7 @@ def run(self):
         "build_ext": build_ext_subclass,
         "build_plugins": BuildPluginsCommand,
     },
+    packages=['invesalius_pubsub'],
```

Comment:
This will not be necessary if move the invesalius_pubsub

Reply:
Done.